### DISCUSSION TITLE
Spin up for running Dynamic vegetation in Noah-MP 4.0.1

### DISCUSSION THREAD

**Discussion by Arijit22IITDelhi** [NCAR/noahmp#132] [Q&A]
Hello, I was running Noah-MP version 4.0.1. I enabled the dynamic vegetation option to check GPP estimates. To get equilibrium carbon balance condition, how long spin up runs are recommended?

**Comment by cenlinhe** [comment]
@CharlesZheZhang

**Comment by keenmisty** [comment]
It depends on whether you consider the carbon accumulation in the soil and the impact of climate conditions. If so, my personal suggestion is to perform approximately 20-100 loops for a while.

**Reply by tslin2** [reply]
Hi,
How many years are in one loop for your Noah-MP runs? 

Thanks.

**Comment by nkianggiss** [comment]
> Hello, I was running Noah-MP version 4.0.1. I enabled the dynamic vegetation option to check GPP estimates. To get equilibrium carbon balance condition, how long spin up runs are recommended?

Hi, reading the Dickinson et al. 1998 interactive growth model, it looks like you need probably 60-100 years to spin up the plant growth to mature canopies, since it takes that long for trees to grow and for succession to happen.  For soil carbon, it might take you > 6000 years.  In the GISS ModelE-Ent vegetation, we spin up the soil carbon sometimes in 2-3 interations of 6750 years, in which each iteration cycles 9 years of meteorological forcings.  This equilibrates the soil carbon, which matches inventory estimates pretty well.  But it will depend on your model

**Reply by tslin2** [reply]
Agreed. The soil requires much more time than GPP to get steady state. I am curious if any previous studies have looked into the soil carbon in Noah-MP?
